crates/napi/src/next_api/project.rs

```diff
@@ -444,15 +473,47 @@ pub async fn project_new(
     ))
 }
 
+#[derive(Debug, Clone, serde::Serialize)]
+struct SlowFilesystemEvent {
+    directory: String,
+    duration: u128,
```

Comment:
I'm guessing you're not using `Duration` here because of `Serialize`, but at least note the unit.

```suggestion
    duration_ms: u128,
```

There's also tricks you can use with serde field attributes to store this as `Duration` and only do the conversion during serialization, but it's probably not worth it: https://serde.rs/field-attrs.html

Reply:
isn't u128 a lot of precision? not a huge deal but that is a lot of milliseconds

Reply:
@lukesandberg `u128` is the default precision for `Duration` when working with ms. I would have to cast to a lower number and lose precision and potentially have errors if I wanted to store less. To answer your question though, yes, it is a lot of precision here haha. For the amount of events we expect to be sending for a slow filesystem though, I don't think it will meaningfully impact our memory usage.

Reply:
@bgw Thanks for the callout, I'll add that in!